homeassistant/components/blink/diagnostics.py

```diff
@@ -0,0 +1,34 @@
+"""Diagnostics support for Aladdin Connect."""
+from __future__ import annotations
+
+from typing import Any
+
+from blinkpy.blinkpy import Blink
+
+from homeassistant.components.diagnostics import async_redact_data
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+TO_REDACT = {"serial", "macaddress", "username", "password", "token"}
+
+
+async def async_get_config_entry_diagnostics(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+) -> dict[str, Any]:
+    """Return diagnostics for a config entry."""
+
+    api: Blink = hass.data[DOMAIN][config_entry.entry_id].api
+
+    data = {}
+    for _, camera in api.cameras.items():
```

Comment:
Looks like this could be a comprehension instead

Reply:
Sure enough - and a good educational experience :)